[32mINFO    [0m logger:test_api.py:536 test delete contact unauthorized successfully complete